.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "npm"
+    directory: "hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "[Recon] Dependabot Package Upgrade: "
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
```

Comment:
Does this mean dependabot will not do major version bumps for any packages? It looks like in the image shared it did a major version bump of `react-router`.

Reply:
Sorry, uploaded the wrong SS, let me update it.
Also for reference here are the PRs raised against my forked repo:
https://github.com/devabhishekpal/ozone/pull/12
https://github.com/devabhishekpal/ozone/pull/13
https://github.com/devabhishekpal/ozone/pull/14
https://github.com/devabhishekpal/ozone/pull/15
https://github.com/devabhishekpal/ozone/pull/17